### PR TITLE
[doc] Minor updates in `hello.rst`

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related issue = #941 

(Found these issues while translating the doc to Chinese)

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

**Comment by yuanming-hu** [comment]
@TH3CHARLie thanks for the tips. Although the code samples in `Interacting with Python` is not part of `fractal.py`, I still think it is helpful to mention these features here. I made minor modifications to make the transition smoother.

**Comment by TH3CHARLie** [comment]
LGTM! The new code samples added in `Interacting with Python` makes this a complete and great introduction now!

File: docs/hello.rst [hunk 1]
```diff
@@ -129,8 +129,8 @@ You can also define Taichi **functions** with ``ti.func``, which can be called a
 
 .. note::
 
-  **Taichi-scope v.s. Python-scope**: everything decorated with ``ti.kernel`` and ``ti.func`` is in Taichi-scope, which will be compiled by the Taichi compiler.
-  Code outside the Taichi-scopes is simply normal Python code.
+  **Taichi-scopes v.s. Python-scopes**: everything decorated with ``ti.kernel`` and ``ti.func`` is in Taichi-scope, which will be compiled by the Taichi compiler.
```
[the remark this answers is not in the document]
Comment (yuanming-hu) [reply docs/hello.rst:132]: Let's unify the usage of `scope` in a separate PR to make this one easier to review :-)

File: docs/hello.rst [hunk 2]
```diff
@@ -129,8 +129,8 @@ You can also define Taichi **functions** with ``ti.func``, which can be called a
...
+  Everything else is in Python-scopes. They are simply Python code.
```
[the remark this answers is not in the document]
Comment (yuanming-hu) [reply docs/hello.rst:133]: It doesn't read good to me either, but at least it's grammatically correct. Let's tolerate these places for now (unless there is a solution that is clearly better). Hopefully, someone native in English can help us out in the future.

**Comment by yuanming-hu** [comment]
Merging this for v0.6.4. As promised I'll unify the terminologies later: https://github.com/taichi-dev/taichi/issues/941#issuecomment-628179633

**Pull request merged**